integration_tests/pages/refer/findPerson.ts

```diff
@@ -1,11 +1,18 @@
 import Helpers from '../../support/helpers'
 import Page from '../page'
+import type { Prisoner } from '@prisoner-offender-search'
 
 export default class FindPersonPage extends Page {
   constructor() {
     super("Enter the person's identifier")
   }
 
+  searchForPrisonerNumber(prisonerNumber: Prisoner['prisonerNumber']) {
+    cy.get('input#prisonNumber').type(prisonerNumber)
+
+    cy.contains('.govuk-button', 'Continue').click()
```

Comment:
You could use the `shouldContainButton` method you wrote here? `this.shouldContainButton('Continue').click()`